modyn/selector/internal/selector_strategies/downsampling_strategies/rs2_downsampling_strategy.py

```diff
@@ -0,0 +1,14 @@
+from modyn.config.schema.sampling.downsampling_config import RS2DownsamplingConfig
+from modyn.selector.internal.selector_strategies.downsampling_strategies import AbstractDownsamplingStrategy
+
+
+class RS2DownsamplingStrategy(AbstractDownsamplingStrategy):
+    def __init__(
+        self,
+        downsampling_config: RS2DownsamplingConfig,
+        modyn_config: dict,
+        pipeline_id: int,
+        maximum_keys_in_memory: int,
+    ):
+        super().__init__(downsampling_config, modyn_config, pipeline_id, maximum_keys_in_memory)
+        self.remote_downsampling_strategy_name = "RS2DownsamplingStrategy"
```

Comment:
Also the name `self.remote_downsampling_strategy_name` should match the remote class name. That's how the remote class is correctly selected and initialized. Relevant code is in `pytorch_trainer.py` feel free to take a look at how it is initialized.

Reply:
Thanks for catching this. I agree we should somehow integration test this, but this is a bigger task since it tests the interaction of two components. Let's do that separately